core/templates/dev/head/base.html

```diff
@@ -51,7 +51,11 @@
     <meta property="og:image" content="{{DOMAIN_URL}}/images/logo/288x288_logo_mint.png">
 
     <link rel="apple-touch-icon" href="/images/logo/favicon.png">
-    <title itemprop="name" ng-bind="pageTitle">{% block subtitle %}{% endblock %} - {% block maintitle %}Oppia{% endblock %}</title>
+    <!-- The title is binded to the rootScope and it will be replaced for its
+    translation when the page is loading. This is the only way to translate the
+    page title because the head of the file is outside the scope of any other
+    controller. -->
```

Comment:
I think using, say, the editor page is fine. If the page is eventually translated, we'll modify the test, but I suspect it will be a long time before the editor page gets translated because there's so much stuff in it.